src/main/java/org/prebid/server/bidder/dmx/DmxBidder.java

```diff
@@ -278,4 +312,11 @@ private static String getAdm(Bid bid) {
         final String wrappedNurl = String.format(IMP, bid.getNurl());
         return bid.getAdm().replaceFirst(SEARCH, wrappedNurl);
     }
+
+    private static List<Integer> checkVideoProtocols(Video video) {
```

Comment:
resolveVideoProtocols, and as a parameter can have List<Integer> protocols not whole video

Reply:
Done